### ISSUE TITLE
avirecorder wont record any frames

### ISSUE THREAD

**Issue by jacktheepic** [AronParker/AviRecorder#16]
for some reason i press f9 to record, it just slightly slow down the video, and in the avirecorder window, the "current movie" and "last frame processed" box is empy and not doing anything. help plz

**Comment by freiz000** [comment]
Post two screenshots of the game's console:

1. After you press F9, and
2. after the demo has finished and you've pressed F9 again.

**Comment by suikaMUSIC** [comment]
I'm also getting the same issue, but the demo speeds up instead of slowing down and I get this in console when I press F9: 

"StartMovie: invalid path: D:\Videos\frag\tga\rec_"

This is the path where I normally record my tga files, but it hasn't worked ever since the recent update.  I also get this in console when I press F9 again to finish:

"No movie started."

Can someone please tell me what's going on?

**Comment by suikaMUSIC** [comment]
also the newer posts in this thread may be of some help: https://github.com/quanticc/lawena-recording-tool/issues/82

apparently the recent update caused absolute paths to not work anymore or something?

**Comment by freiz000** [comment]
Try putting the AviRec exe in the folder where startmovie normally (without the use of AviRec) creates the images/audio. And then in the AviRec settings, instead of having the TGA/AVI paths to "C:\etc\etc..." just manually type "tga" and "avi. Be sure to have created those folders first.

**Comment by suikaMUSIC** [comment]
Bless you, it worked.

**Comment by Optivex** [comment]
Yep the update just doesn't like custom file paths in the startmovie command here's some detailed steps on how to fix it using the default path:
In AviRecorder set the TGA folder to your tf2 root tf folder (for me it's C:\Program Files (x86)\Steam\steamapps\common\Team Fortress 2\tf). Avi folder can still be whatever you want obviously.
When when avirecorder launches go into the tf/cfg folder and edit avi_recorder_start.cfg
On the second line it should say something like this: startmovie "C:\Program Files (x86)\Steam\steamapps\common\Team Fortress 2\tf\"
Change it to this: startmovie rec_ 
you could save this change by making your own avirecorder config
